src/backends/interflop-mca-int/interflop_mca_int.c

```diff
@@ -393,22 +388,20 @@ error_t parse_opt(int key, char *arg, struct argp_state *state) {
     /* precision for binary32 */
     errno = 0;
     val = strtol(arg, &endptr, 10);
-    if (errno != 0 || val <= 0) {
-      logger_error("--%s invalid value provided, must be a positive integer",
+    if (errno != 0 || val != MCA_PRECISION_BINARY32_DEFAULT) {
+      logger_error("--%s invalid value provided, MCA integer does not support "
+                   "custom precisions",
                    key_prec_b32_str);
-    } else {
```

Comment:
So you remove the opportunity for the user to use virtual prec. But your are also forbiding it for an expert user that could live with some bias under certain circumstance?  Can't we have an hidden parameter and keep the code?

Reply:
I think it's better to forbid it in MCA int. The result distribution is biased when using a precision different than the working one; and the operation is different than what one would expect for MCA RR.

 If an expert user still wants to play with it, it's pretty trivial to switch it back, don't you think ? Better to add it back then, than keep the complexity of a hidden option ?

Reply:
Btw, I keep the option in MCA quad. So it's still possible to have virtual precision in MCA (just with a different backend).

Reply:
It is true that we never used so much virtual precision in mca since we have vprec for the precision work. In  the reare case we would like both I suppose mca quad is fast enough, and if not then we might pay the price of having this hidden option (like --force or something to force it to be accepted).